modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+== Locating the encapsulated namespaces
```

Comment:
I think we should rename this section "Running additional services in the encapsulated namespace" and move it AFTER the "Inspecting the encapsulated namespace" section below.